### PR TITLE
fix: Correct mis-spelling in serverless security policy default description

### PR DESCRIPTION
**Pull request by dim-ops**
## Description
Spelling error. Newtwork => network

### CODE REVIEW

File: modules/collection/main.tf [hunk 1]
```diff
@@ -64,7 +64,7 @@ resource "aws_opensearchserverless_security_policy" "encryption" {
 resource "aws_opensearchserverless_security_policy" "network" {
   count = var.create && var.create_network_policy ? 1 : 0
 
-  description = coalesce(var.network_policy_description, "Newtwork policy for ${var.name} collection")
```
Comment (bryantbiggs) [inline modules/collection/main.tf:67]: I don't know - `Newtwork` sounds kinda cool 😅

**Review by bryantbiggs — APPROVED** [review]
thank you!

**Comment by antonbabenko** [comment]
This PR is included in version 1.2.2 :tada:

**Comment by dim-ops** [comment]
@antonbabenko I hesitate to take the time to add the creation of the VPC endpoint to the module, what do you think? https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/opensearch_vpc_endpoint

**Comment by bryantbiggs** [comment]
That would go in the VPC module

**Pull request merged**